src/components/New/Calendar.tsx

```diff
@@ -95,7 +96,7 @@ const defaultOptions = (
     day_today: "text-spacePurple-300 hover:text-white ",
   },
   components: {
-    Caption: mode === "range" ? CustomCaptionSided : CustomCaptionGrouped,
+    Caption: mode === "range" && (numberOfMonths || 1) > 1 ? CustomCaptionSided : CustomCaptionGrouped,
```

Comment:
Don't think you need a fallback
```suggestion
    Caption: mode === "range" && (numberOfMonths) > 1 ? CustomCaptionSided : CustomCaptionGrouped,
```